pkgs/development/libraries/qt-5/modules/qtmultimedia.nix

```
@@ -8,8 +8,9 @@ qtModule {
   name = "qtmultimedia";
   qtInputs = [ qtbase qtdeclarative ];
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ gstreamer gst-plugins-base libpulseaudio]
-    ++ optional (stdenv.isLinux) alsaLib;
+  buildInputs = [ gstreamer gst-plugins-base libpulseaudio ]
+    ++ optional (stdenv.isLinux) alsaLib
+    ++ optional (versionAtLeast qtbase.version "5.14.0") wayland;
```

```suggestion
    ++ optional (versionAtLeast qtbase.version "5.14.0" && stdenv.isLinux) wayland;
```

I think this should clear up the eval error.

---

Thanks!